CMakeLists.txt

```diff
@@ -159,20 +159,45 @@ if (WIN32)
     add_definitions(-DNOMINMAX -DWIN32_LEAN_AND_MEAN)
 endif()
 
-# Dependencies
+if (ANDROID)
+    set(CMAKE_FIND_ROOT_PATH ${OPENMW_DEPENDENCIES_DIR} "${CMAKE_FIND_ROOT_PATH}")
+    set(OSG_PLUGINS_DIR ${OSG_PLUGINS_DIR})
+    add_definitions (-DOSG_PLUGINS_DIR)
```

Comment:
This definition seems to be unused. Maybe you are confusing definitions with variables? add_definitions affects the compiling of code. set(VAR VALUE) affects the cmake script.

Also, if you wanted to pass the plugin directory to the code, you'd need to add a value to the definition as well e.g.
<pre>
add_definitions (-DOSG_PLUGINS_DIR=${OSG_PLUGINS_DIR})
</pre>